.github/workflows/monitor-stable-releases.yml

```diff
@@ -0,0 +1,52 @@
+---
+name: Monitor-stable-releases
+
+#on:
+#  release:
+#    types: [released]
```

Comment:
Are these supposed to be commented out?

Also, we probably want a workflow dispatch option here so we can manually trigger this.

Reply:
No I left them commented out as a precaution, I also use the ref/master but I wanted to make sure that nothing gets published before we review this

Reply:
> Also, we probably want a workflow dispatch option here so we can manually trigger this.

yes we can